srcpkgs/pcsxr-pgxp/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'pcsxr-pgxp'
+pkgname=pcsxr-pgxp
+version=1.9.99
+revision=1
+lib32disabled=yes
+wrksrc="pcsxr-master"
+nostrip_files="pcsxr"
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_INSTALL_LIBDIR=/usr/lib
+ -DSND_BACKEND=sdl -DENABLE_CCDDA=ON -DUSE_LIBARCHIVE=ON -DUSE_LIBCDIO=ON -DOpenGL_GL_PREFERENCE=GLVND"
+hostmakedepends="pkg-config automake libtool intltool glib-devel gettext-devel nasm zip unzip"
+makedepends="libXv-devel libXtst-devel libcdio-devel SDL-devel gtk+3-devel libarchive-devel
+ libXv-devel gdk-pixbuf-devel SDL2-devel SDL-devel libXext-devel libpulseaudio libX11-devel libXxf86vm
+ zlib-devel libepoxy-devel glu-devel MesaLib-devel ffmpeg ffmpeg-devel"
+depends="desktop-file-utils ffmpeg libarchive libcdio libXv gdk-pixbuf libGL libGLES SDL2 SDL libXext
+ libpulseaudio libX11 libXxf86vm zlib libepoxy glu libtxc_dxtn libavcodec libavdevice libavformat libavfilter
+ libavresample libswresample libswscale libavutil"
+short_desc="Sony PlayStation (PSX) emulator based on the PCSX-df project"
+maintainer="VladTheImpaler <vladimir-romanov-ii@yandex.com>"
+license="GPL-3"
+homepage="http://pcsxr.codeplex.com/"
+distfiles="https://github.com/icatbutler/pcsxr/archive/master.zip"
```

Comment:
This is wrong! You should never fetch the master zip from github.
Use the release tarball, or a precise commit if tarball is not available.

Reply:
There is no release tarball... :/ is why I use master.zip

Reply:
Are you sure about `version=1.9.99` ?
According to [ChangeLog](https://github.com/iCatButler/pcsxr/blob/master/ChangeLog) the latest seems to be 1.9.92.

Reply:
You can fetch the [latest commit](https://github.com/icatbutler/pcsxr/commit/7936d466c58f6f2603900fbbc9d18b56ac2e4b4c) by modifying your template:
````
version=1.9.92.20180427
_commit=7936d466c58f6f2603900fbbc9d18b56ac2e4b4c
wrksrc="pcsxr-${_commit}"

distfiles="https://github.com/icatbutler/pcsxr/archive/${_commit}.tar.gz"
checksum=5d7541abef8f8f42ca92bee4f7f6785bd8ea2dcf8eed65261cd13de798dfbe13
````
Should work.